auto-update-pip-packages/update-openfisca-virtualenv.sh

```diff
@@ -1,40 +1,17 @@
-#!/bin/bash
+#!/usr/bin/env bash
 
+# Adapt this variable to your server.
+VENV_ACTIVATE="/home/openfisca/virtualenvs/openfisca/bin/activate"
 
-# Customizable variables
-PIP_PACKAGES="OpenFisca-France OpenFisca-Core OpenFisca-Web-API"
-VENV_ACTIVATE="$HOME/virtualenvs/openfisca/bin/activate"
-# End of customizable variables
-
-
-CURL="curl --silent --retry 5"
-restart_services="0"
-
-function upgrade_package {
-    PACKAGE_NAME="$1"
-    CURRENT_VERSION=`python -c "import pkg_resources; print(pkg_resources.get_distribution('$PACKAGE_NAME').version)"`
-    pip install --quiet --no-cache-dir --upgrade $PACKAGE_NAME
-    NEW_VERSION=`python -c "import pkg_resources; print(pkg_resources.get_distribution('$PACKAGE_NAME').version)"`
-    if [[ "$NEW_VERSION" != "$CURRENT_VERSION" ]]; then
-        echo "- New version of package $PACKAGE_NAME was found on PyPI: current=$CURRENT_VERSION, new=$NEW_VERSION"
-        restart_services="1"
-    fi
-}
-
-source $VENV_ACTIVATE
+set -x
+source "$VENV_ACTIVATE"
+pip install --requirement /home/openfisca/openfisca-ops/auto-update-pip-packages/requirements.txt --upgrade
```

Comment:
To make sure the `requirements.txt` is in its latest version, download it from github first.